hbase-shaded-miscellaneous/pom.xml

```diff
@@ -89,6 +89,38 @@
                   <pattern>org.apache.commons</pattern>
                   <shadedPattern>${rename.offset}.org.apache.commons</shadedPattern>
                 </relocation>
+                <relocation>
+                  <pattern>org.eclipse.jetty</pattern>
+                  <shadedPattern>${rename.offset}.org.eclipse.jetty</shadedPattern>
+                </relocation>
+                <relocation>
+                  <pattern>javax</pattern>
```

Comment:
this is dangerous because the javax package is special within the JDK specs. classes in this package should be part of the JDK or a JSR, so ideally we wouldn't need to shade them.

what is this trying to relocate?

Reply:
thanks for the comment!
there are a bunch of javax.* classes in jetty and glassfish for the reason i don't know. maybe they are shaded in jetty/glassfish but they don't relocate them? 

would you suggest i ignore these javax classes in hbase?

Reply:
Let me put aside some time to see what they are. Maybe middle to end of this week?

My guess would be they're either JSR for web services or some related JEE thing.

Reply:
sure no worries. This work is only meant make the community understand the scope of change, and whether or not we should do it.